Source/Device.generated.swift

```diff
@@ -561,6 +561,14 @@ public enum Device {
       return (width: -1, height: -1)
     #endif
   }
+    
+  #if os(iOS) || os(tvOS)
```

Comment:
I think we could even add this for watchOS, though I am not sure.

```suggestion
  #if os(iOS) || os(tvOS) || os(watchOS)
```